Misc/NEWS.d/next/Core and Builtins/2023-03-30-15-29-55.gh-issue-103131.pFNxUN.rst

```diff
@@ -0,0 +1,2 @@
+Convert :func:`sys.getsizeof` and :func:`sys.set_asyncgen_hooks`
```

Comment:
It is an internal refactoring. Such changes usually are not reflected in the NEWS file.